Create zh_tw.json

HJ-zhtw opened the pull request.

Adubbz committed:
Updated for the water purifier

VDeltaGabriel committed:
Polish Transltion (#682)

Polish translation for mod

mindy15963 committed:
Create ko_kr.json (#686)

* Create ko_kr.json

* Update ko_kr.json

omerfaruk-cakmak committed:
Create tr_tr.json (#689)

oszoou committed:
Update pl_pl.json (#690)

Covariance committed:
Create ru_ru.json (#695)

Russian translation added

paladini-maker committed:
Create es_es.json (#700)

Added support for Spanish language

Samlegamer committed:
French Translation (#701)

Complete translation of the Mod into French.

HJ-zhtw committed:
Create zh_tw.json

Adubbz commented:
I updated the repo for 1.17, so you may need to resubmit this I'm afraid. It previously was still reflecting the 1.16 version

HJ-zhtw committed:
Update zh_tw.json

Updated to 1.17.

HJ-zhtw commented:
> I updated the repo for 1.17, so you may need to resubmit this I'm afraid. It previously was still reflecting the 1.16 version

Okay, just updated to 1.17.

Adubbz commented:
It looks like there's still a bunch of files that conflict so it won't allow me to merge your PR. Could you reclone the repo, add your translation, then resubmit?
